src/main/java/uk/gov/di/ipv/persistence/item/AuthorizationCodeItem.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.di.ipv.persistence.item;
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+public class AuthorizationCodeItem {
```

Comment:
I'm wondering if we might change this to be a more generic item (and table) in future. Rather than maintaining a table just for auth codes, we might end up with the access token and maybe the user_id from Auth&Orc in there. Maybe some other bits. I guess we can change this if/when we get there though.